packages/orm/src/EntityManager.ts

```diff
@@ -233,33 +240,21 @@ export class EntityManager<C = unknown, Entity extends EntityW = EntityW, TX ext
   private __isRefreshing = false;
   mode: EntityManagerMode = "writes";
 
-  constructor(em: EntityManager<C, Entity, TX>);
   constructor(ctx: C, opts: EntityManagerOpts<TX>);
   constructor(ctx: C, driver: Driver<TX>);
-  constructor(emOrCtx: EntityManager<C, Entity, TX> | C, driverOrOpts?: EntityManagerOpts<TX> | Driver<TX>) {
-    if (emOrCtx instanceof EntityManager) {
-      // Setup from the existing EntityManager
-      const em = emOrCtx;
-      this.driver = em.driver;
-      this.#preloader = em.#preloader;
+  constructor(ctx: C, driverOrOpts: EntityManagerOpts<TX> | Driver<TX>) {
```

Comment:
note: the old behavior of allowing the creation of an `em` from another `em` was, imo, a footgun due to the way it interacted with `ctx`.